apps/condo/domains/notification/schema/SyncDeviceService.js

```diff
@@ -19,7 +18,7 @@ const SyncDeviceService = new GQLCustomSchema('SyncDeviceService', {
         },
         {
             access: true,
-            type: 'input SyncDeviceInput { dv: Int!, sender: SenderFieldInput!, deviceId: String!, pushToken: String, pushTransport: PushTransportType, meta: JSON }',
+            type: 'input SyncDeviceInput { dv: Int!, sender: SenderFieldInput!, deviceId: String!, pushToken: String, pushTransport: PushTransportType!, meta: JSON }',
```

Comment:
you can sync device without pushToken

Reply:
It's pushTransport, not token
there could be deviceId collision between different transport systems, so deviceId should always come together with pushTransport